Svc/Interfaces/ComInterface.fppi

```diff
@@ -1,8 +1,22 @@
-@ Data coming in from the framing component
-sync input port comDataIn: Drv.ByteStreamSend
+# ----------------------------------------------------------------------
+# Com Data and Status
+# ----------------------------------------------------------------------
+
+@ Data coming in to be sent out for transmission (e.g. from a Framer)
+sync input port comDataIn: Svc.ComDataWithContext
+
+@ Buffers sent out for transmission (e.g. to a driver)
+output port comDataOut: Drv.ByteStreamData
```

Comment:
comDataOut (to driver) and dataReturnIn  should not be in this interface.  They are a function of ComStub wrapping a driver.  If you have a single radio implementation that is one component, it does not have output ports.

Reply:
Fix in this PR.

Reply:
comDataOut is actually going to FrameAccumulator (for uplink), the annotation is wrong. FrameAccumulator used to take in Drv.ByteStreamRecv which I suggest we should change to have to go through a ComInterface. Fw.BufferSend is likely appropriate. Will fix.